### PR TITLE
Add php-cs check action

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/mail/runs/586187169 for an example of a failed check

**Comment by kesselb** [comment]
Test here is also failing :)

**Review by kesselb — APPROVED** [review]

**Review by gary-kim — COMMENTED** [review]
Could be nice to have a message in the workflow run that says issues can be fixed by running `composer run cs:fix`. What do you think?

**Review by nickvergessen — APPROVED** [review]

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -18,3 +18,19 @@ jobs:
...
+      uses: actions/checkout@master
+    - name: Set up php
+      uses: shivammathur/setup-php@master
+      with:
+        php-version: 7.4
+        coverage: none
+    - name: Install dependencies
+      run: composer i
+    - name: Run coding standards check
+      run: composer run cs:check || ( echo 'Please run `composer run cs:fix` for format your code' && exit 1 )"
```
Comment (raimund-schluessler) [inline .github/workflows/lint.yml:36]:
```suggestion
      run: composer run cs:check || ( echo 'Please run `composer run cs:fix` to format your code' && exit 1 )"
```

**Review by gary-kim — APPROVED** [review]

**Pull request merged**